python/pyspark/sql/utils.py

```diff
@@ -144,7 +144,7 @@ def is_remote() -> bool:
     """
     Returns if the current running environment is for Spark Connect.
     """
-    return "SPARK_REMOTE" in os.environ
+    return "SPARK_ENABLE_CONNECT_MODE" in os.environ
```

Comment:
IIRC, we will drop the usage of `is_remote` ?  @HyukjinKwon

Reply:
No, the usage remains the same.